website/content/docs/auth/azure.mdx

```diff
@@ -21,6 +21,51 @@ This documentation assumes the Azure method is mounted at the `/auth/azure`
 path in Vault. Since it is possible to enable auth methods at any location,
 please update your API calls accordingly.
 
+## Token validation ((#token-validation))
+
+Vault validates the resource group (`resource_group_name`), VM name (`vm_name`)
+and VM scale set name (`vmss_name`) parameters against token claims. Depending
+on the identities attached to the machine generating the MSI token, the
+associated claim must include at least one of the following claims
+to pass validation: `xms_mirid` or `xms_az_rid`.
+
+Vault also expects specific metadata and identity information depending on your
+system configuration.
+
+System assigned managed identity | User-assigned identity| Identity specified in Azure managed identity REST endpoint
+-------------------------------- | ----------------------| ----------------------------------------------------------
+Enabled                          | Any                   | Not required
+Not enabled                      | 1                     | Not required
+Not enabled                      | Multiple              | Required
+
+See [Azure managed identity REST endpoint reference](https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp#rest-endpoint-reference)
+for more details on how to specify the identity in the request.
+
+#### When System-assigned managed identity is enabled
```

Comment:
Why are we breaking related information across 2 tables and introducing headings here?